deploy/osd-oauth-templates-errors/osd-oauth-templates-errors.secret.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: v1
```

Comment:
Why is this (and all the other `osd-oauth` files) being removed as part of this PR?

Reply:
Cleared up unwanted deleted files